src/tests/ascent/t_ascent_mpi_vtk_file_extract.cpp

```diff
@@ -0,0 +1,224 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) Lawrence Livermore National Security, LLC and other Ascent
+// Project developers. See top-level LICENSE AND COPYRIGHT files for dates and
+// other details. No copyright assignment is required to contribute to Ascent.
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+//-----------------------------------------------------------------------------
+///
+/// file: t_ascent_mpi_vtk_file_extract.cpp
+///
+//-----------------------------------------------------------------------------
+
+
+#include "gtest/gtest.h"
+
+#include <ascent.hpp>
+#include <mpi.h>
+
+#include <iostream>
+#include <math.h>
+
+#include <conduit_blueprint.hpp>
+#include <conduit_relay.hpp>
+
+#include "t_config.hpp"
+#include "t_utils.hpp"
+
+
+using namespace std;
+using namespace conduit;
+using namespace ascent;
+
+
+index_t EXAMPLE_MESH_SIDE_DIM = 10;
+
+//-----------------------------------------------------------------------------
+TEST(ascent_vtk_file_extract, basic_mulit_domain)
```

Comment:
The mulit domain -- structured in the front, unstructured in the back ;)

Reply:
now I have to keep the typo :-)